web/views.py

```
@@ -6955,3 +6973,448 @@ def delete_post(request, post_id):
+
+    context = {
+        "plan": plan,
+        "billing_period": billing_period,
+        "price": price,
+        "stripe_key": stripe_key,
+    }
+
+    return render(request, "membership/checkout.html", context)
+
+
+def change_membership_plan(request: HttpRequest) -> HttpResponse:
```

Ruff (0.8.2)

Unused function argument: `request`

(ARG001)